packages/frontend/src/components/ProjectConnection/WarehouseForms/SnowflakeForm.tsx

```diff
@@ -164,6 +170,18 @@ const SnowflakeForm: FC<{
               },
           ];
 
+    // Only show EXTERNAL_BROWSER if it's already selected (edit mode only)
```

Comment:
How would we get the External Browser option if it's not already selected? Would that be that it was created and set in CLI?